Add blog post: LibrePCB Meets FontoBene

ubruhin opened the pull request.

dbrgn commented on content/blog/2018-04-19_librepcb_meets_fontobene.md:
```diff
@@ -0,0 +1,61 @@
+
+One reason is that OpenType and TrueType fonts are very complex and thus require
+high effort to integrate them into LibrePCB. For example the Gerber export is
+not as trivial as you may expect because Gerber doesn't support fonts. On the
```
What about replacing "fonts" with "Bézier curves"?

dbrgn commented on content/blog/2018-04-19_librepcb_meets_fontobene.md:
```diff
@@ -0,0 +1,61 @@
+
+### Introducing FontoBene
+
+Unfortunately we were not happy with the available stroke fonts because of
```
fonts -> font formats?

dbrgn commented on content/blog/2018-04-19_librepcb_meets_fontobene.md:
```diff
@@ -0,0 +1,61 @@
+there aren't any clear specifications available and the existing fonts have
+non-permissive, partly even unknown licenses.
+
+![FontoBene](/img/fontobene.png)
```
Hm, you're inserting this screenshot just after the section where you complain about missing arcs, but the example font clearly consists of only straight lines :)

dbrgn replied:
Yeah, that's probably the easiest solution. Maybe between the two paragraphs in the outlook section?

dbrgn commented on content/blog/2018-04-19_librepcb_meets_fontobene.md:
```diff
@@ -0,0 +1,61 @@
+
+Currently LibrePCB uses the [NewStroke](https://vovanium.ru/sledy/newstroke/en)
+font which we converted to FontoBene, mainly because it is available under the
+CC0 Public Domain license. So we don't yet use the advantages of having support
```
So we don't yet use the advantages -> This means that we currently don't yet take advantage

dbrgn commented:
@ubruhin if you use "commit --fixup" instead of amending, then it's easier to see the changes in the pull request history :) you can always squash before / when merging.

ubruhin commented:
@dbrgn I know, sorry for not doing it ;) I rebased this branch onto master because I fixed the broken CI first, and then I squashed the two commits I've made on this branch...

Now I replaced the FontoBene image with a font which uses arcs, so now one can see how it could look like. In addition, I added a new chapter about embedding fonts into projects because I think this could be an interesting information for some users.

ubruhin commented:
@dbrgn ready to publish?
